File: tests-java/src/main/java/pesoklp13/examples/tests/dummy/app/DummyApplication.java
package pesoklp13.examples.tests.dummy.app;

import org.springframework.boot.Banner;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication(scanBasePackages = "pesoklp13.examples.tests.dummy")
public class DummyApplication {

    public static void main(String[] args) {
        new SpringApplicationBuilder(DummyApplication.class).bannerMode(Banner.Mode.OFF).run(args);
    }
}

File: tests-kotlin/src/main/kotlin/pesoklp13/examples/tests/dummy/app/DummyApplication.kt
package pesoklp13.examples.tests.dummy.app

import org.springframework.boot.Banner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class DummyApplication {

    companion object {
        @JvmStatic fun main(args: Array<String>) {
            runApplication<DummyApplication>(*args) {
                setBannerMode(Banner.Mode.OFF)
            }
        }
    }

}